ckanext/datapusher/logic/action.py

```diff
@@ -53,6 +54,15 @@ def datapusher_submit(context, data_dict):
 
     user = p.toolkit.get_action('user_show')(context, {'id': context['user']})
 
+    for plugin in p.PluginImplementations(interfaces.IDataPusher):
+        upload = plugin.can_upload(res_id)
+        if not upload:
+            msg = p.toolkit._("Plugin {} rejected resource {}"\
```

Comment:
The {} formatters won't work on Python 2.6 (see Travis failure). You need to use {0}.